### DISCUSSION TITLE
Admin commands

### DISCUSSION THREAD

**Discussion by Ynfuien** [BG-Software-LLC/SuperiorSkyblock2#753] [Pending Feature Requests]
In my opinion `/is admin` are missing some really helpful commands.

**-** `/is admin showsettings <player>` - to show player island settings. Currently it's possible to only change settings, but not to show these.
**-** `/is admin showpermissions <player>` - like with settings, to show island permissions.
**-** `/is admin addtoisland(?) <player> <player>` - to be able to add a player to another player's island, cause at this time, I must to play with /sudo command and it only works for online players.
**-** `/is admin createisland <player>` - to create island for player

**Comment by Skizzles** [comment]
@Ynfuien /is admin add (name) (name) can accomplish the addtoisland. However I agree with the other commands :)
